prefixcommons/curie_util.py

```diff
@@ -100,9 +102,14 @@ def read_biocontext(name: str) -> PREFIX_MAP:
 ]
 
 
+default_converter = curies.chain(
+    Converter.from_prefix_map(prefix_map) for prefix_map in default_curie_maps
+)
+
+
 def get_prefixes(cmaps: Optional[List[PREFIX_MAP]] = None) -> List[str]:
     if cmaps is None:
-        cmaps = default_curie_maps
+        return sorted(default_converter.get_prefixes())
```

Comment:
Hi @cthoyt - does this change the order of the prefix maps from what they were before?

Reply:
It appeared that the existing order was random (or was based on implicit assumptions about python data structures) so I don't think that there's a satisfying answer to your question.

Reply:
Additionally, it could have been the case that prefixes were duplicated since the logic was to just extend a list. That doesn't make a lot of sense to me either

Reply:
Agree; we can definitely make the code better.   Is there a problem you are trying to solve for the `sorted` addition?

Reply:
the `curies` package returns a set, which seems more meaningful since this doesn't have an inherent ordering. However, this package expects a list so sorting the list to make it deterministic seemed reasonable

Reply:
makes good sense, however, if we leave off the `sorted` we might have a better chance of preserving the serendipitous effects here?  (the worry for me is downstream dependencies that depend on a specific ordering; I've had cases where I've tried to reorder the many different contexts to take advantage of one or the other's more complete or updated maps, only to find it breaks project assumptions about prefixes, etc.).   Tangential to this PR perhaps, but it would be good to know our strategy for aligning this codebase with bioregistry.  https://github.com/prefixcommons/prefixcommons-py/issues/24

Reply:
so rather than `sorted` just do `list`? That's fine for me if you think it will work

Reply:
I made this update in 4987c5518b3540100d49fd9affaec015cad26525